<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTrackReportClass</class>
 <widget class="QWidget" name="CreateTrackReportClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateTrackReport</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Track Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>From : </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fromDt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>46</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="fromTM">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>46</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>hh:mm</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="toDt">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>47</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>51</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>To : </string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="toTM">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>47</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>hh:mm</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
